controllers/depresolver/depresolver_config.go

```diff
@@ -212,6 +209,64 @@ func (dr *DependencyResolver) validateConfig(config *Config, recognizedDNSTypes
 	return nil
 }
 
+func validateConfigForInfoblox(config *Config) error {
+	err := field(InfobloxGridHostKey, config.Infoblox.Host).matchRegexps(hostNameRegex, ipAddressRegex).err
+	if err != nil {
+		return err
+	}
+	err = field(InfobloxVersionKey, config.Infoblox.Version).isNotEmpty().matchRegexp(versionNumberRegex).err
+	if err != nil {
+		return err
+	}
+	err = field(InfobloxPortKey, config.Infoblox.Port).isHigherThanZero().isLessOrEqualTo(65535).err
+	if err != nil {
+		return err
+	}
+	err = field(InfobloxUsernameKey, config.Infoblox.Username).isNotEmpty().err
+	if err != nil {
+		return err
+	}
+	err = field(InfobloxPasswordKey, config.Infoblox.Password).isNotEmpty().err
+	if err != nil {
+		return err
+	}
+	err = field(InfobloxHTTPPoolConnectionsKey, config.Infoblox.HTTPPoolConnections).isHigherOrEqualToZero().err
+	if err != nil {
+		return err
+	}
+	err = field(InfobloxHTTPRequestTimeoutKey, config.Infoblox.HTTPRequestTimeout).isHigherThanZero().err
+	if err != nil {
+		return err
+	}
+	return nil
+}
+
+func (dr *DependencyResolver) GetDeprecations() (deprecations []string) {
+	type oldVar = string
+	type newVar struct {
+		Name string
+		Msg  string
+	}
+
+	var deprecated = map[oldVar]newVar{
+		EdgeDNSServerKey: newVar{
+			Name: EdgeDNSServersKey,
+			Msg:  "Pass the hostname or IP address as comma-separated list",
+		},
+		EdgeDNSServerPortKey: newVar{
+			Name: EdgeDNSServersKey,
+			Msg:  "Port is an optional in the comma-separated list of dns edge servers, in following form: dns1:53,dns2",
```

Comment:
does it default to `53` in case of `dns2`?  if yes, then the message is not totally clear

Reply:
added better message here and also to `values.yaml`